src/main/java/eu/greev/dcbot/ticketsystem/service/Transcript.java

```diff
@@ -1,135 +1,114 @@
-                        } catch (ArrayIndexOutOfBoundsException e) { return; }
-                        writer.write(log);
-                        writer.newLine();
-                    } else {
-                        writer.write(line);
-                        writer.newLine();
+                builder.append("[").append(message.getAuthor()).append("] ");
+                List<Edit> edits = message.getEdits();
+                if (!edits.isEmpty()) {
+                    builder.append(message.getOriginalContent()).append(" | Edits:");
+
+                    for (int i = 0; i < edits.size(); i++) {
+                        builder.append(" ").append(edits.get(i).edit());
+
+                        if (edits.size() - 1 != i) {
+                            builder.append(" ->");
+                        }
                     }
-                    transcript.delete();
-                    temp.renameTo(new File("./Tickets/transcripts/" + id + ".txt"));
                 }
-            }
-        } catch (IOException e) {
-            log.error("Could not read transcript of ticket #" + id, e);
-        }
-    }
 
-    public File clean() {
-        File temp = new File("./Tickets/transcripts/" + id + ".temp");
-        try {
-            temp.createNewFile();
-            try (BufferedWriter writer = new BufferedWriter(new FileWriter(temp, true));
-                 BufferedReader reader = new BufferedReader(new FileReader(transcript));) {
-                List<String> lines = reader.lines().toList();
-
-                for (String line : lines) {
-                    if (lines.get(1).equals(line)) continue;
-                    if (lines.get(0).equals(line)) {
-                        writer.write("Transcript of ticket: #" + id);
-                        writer.newLine();
-                        continue;
-                    }
-                    if (!line.contains("} ")) {
-                        writer.write(line);
-                        writer.newLine();
-                        continue;
-                    }
-                    String content = line.split("} ")[1];
-                    writer.write(content);
-                    writer.newLine();
+                if (message.isDeleted() && message.getId() != 0) {
+                    builder.append("~~").append(message.getOriginalContent()).append("~~");
                 }
+                writer.write(builder.toString());
+                writer.newLine();
             }
-            transcript.delete();
-            temp.renameTo(transcript);
         } catch (IOException e) {
-            log.error("Could not clean transcript of ticket #" + id, e);
+            log.error("Could not clean transcript of ticket #" + ticketId, e);
         }
         return transcript;
     }
+
+    private String formatTimestamp(long timestamp) {
+        return new SimpleDateFormat("[hh:mm:ss a '|' yyyy-MM-dd] ").format(Date.from(Instant.ofEpochSecond(timestamp)));
```

Comment:
```suggestion
        return new SimpleDateFormat("[HH:mm:ss '|' yyyy-MM-dd] ").format(Date.from(Instant.ofEpochSecond(timestamp)));
```
I'm thinking maybe this format may be better?

Reply:
I thought the 24h system isnt that common?

Reply:
it depends on what you want to do with it.
i think 24 hour system should be good enough since it's easy to understand

but more importantly are users able to see this formatted string too?
if so then i'd recommend adding the utc timezone too

Reply:
`"yyyy-MM-dd '|' HH:mm:ssZ"` like that?

Reply:
sure

Reply:
The 24h format is the most common and accessible for everyone, please use it.
As for the format, I would suggest yyyy-MM-dd HH:mm:ssZ as it is the one most will intuitive understand when seeing a year in front. If you used any other format, users from the US might think it's using MM-dd while others will think it's dd-MM.